circuit-benchmarks/src/bench_params.rs

```diff
@@ -0,0 +1,3 @@
+pub(crate) const DEGREE: usize = 11;
+pub(crate) const MEMORY_ADDRESS_MAX: usize = 2000;
+pub(crate) const STACK_ADDRESS_MAX: usize = 1300;
```

Comment:
This file should not be commited. As it gets generated with build.rs

Reply:
oops! Should I add it to .gitignore?

Reply:
oops! Should I add it to .gitignore?

Reply:
https://github.com/appliedzkp/zkevm-circuits/pull/238#pullrequestreview-834956937 

If you approve we can merge it!